rel-sign.sh

```diff
@@ -325,6 +360,22 @@ sign() {
 	infop "Signatures are stored in files %s and %s.\n" "v${VERSION}.tar.gz.asc" "v${VERSION}.zip.asc";
 }
 
+verify-remote() {
+	# Validate hashes first
```

Comment:
Is `validate hashes first` for future? As doesn't see hash verification yet :)

Reply:
Good catch, thanks! :p